src/3_branch.js

```diff
@@ -285,15 +284,9 @@ Branch.prototype['init'] = function(app_id, options, callback) {
  * immediately, otherwise, it will return once Branch has been initialized.
  * ___
  */
-Branch.prototype['data'] = function(callback) {
-	if (!callback) { return; }
-
-	var self = this;
-	this._queue(function(next) {
-		callback(null, utils.whiteListSessionData(utils.readStore(self._storage)));
-		next();
-	});
-};
+Branch.prototype['data'] = wrap(2, function(done) {
+	if (typeof done == "function") { done(null, utils.whiteListSessionData(utils.readStore(this._storage))); }
```

Comment:
yeah let's remove it everywhere. part of the reason to have the wrap is to allow it